cmd/sptool/actor.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"os"
+	"path"
+
+	"github.com/BurntSushi/toml"
+	"github.com/mitchellh/go-homedir"
+	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/go-address"
+
+	lapi "github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/cli/spcli"
+)
+
+var actorCmd = &cli.Command{
+	Name:  "actor",
+	Usage: "Manage Filecoin Miner Actor Metadata",
+	Subcommands: []*cli.Command{
+		spcli.ActorSetAddrsCmd(SPTActorGetter),
+		spcli.ActorWithdrawCmd(SPTActorGetter),
+		spcli.ActorRepayDebtCmd(SPTActorGetter),
+		spcli.ActorSetPeeridCmd(SPTActorGetter),
+		spcli.ActorSetOwnerCmd(SPTActorGetter),
+		addTomlArg(spcli.ActorControlCmd(SPTActorGetter, getControlAddresses)),
```

Comment:
This wrapping should be removed and added to `lotus-miner` implementation. We would need to get these local Control list from DB. Perhaps a flag to get DB access and config layer access?

Reply:
Agree, commands like the control ones which need access to config should only be on lotus-miner. A separate version which only works with chain data can be in sptool, but that one only would need to do basic listing and setting.

Reply:
All set. I put a simplified version into SPTool without most roles.